amazon-ivs-react-native.podspec

```diff
@@ -19,13 +19,13 @@ Pod::Spec.new do |s|
   s.license      = package["license"]
   s.authors      = package["author"]
 
-  s.platforms    = { :ios => "10.0" }
+  s.platforms    = { :ios => "11.0" }
```

Comment:
With the upgrade to 1.7.0, IVS bumped minimum OS requirements to iOS 11:
https://docs.aws.amazon.com/ivs/latest/userguide/release-notes.html